src/locales/polyglot.js

```diff
@@ -30,19 +30,22 @@ const extendPolyglot = (locale, polyglot, phrases, mobilePhrases) => {
   return polyglot
 }
 
-const findMissingKeys = (defaultKeys, customKeys) => {
+const findMissingKeys = (defaultKeys, customKeys, language) => {
   const newTranslationsSet = new Set(customKeys)
   const missingKeys =  defaultKeys.filter(element => !newTranslationsSet.has(element))
-  if (missingKeys.length) { console.warn('Missing keys:', missingKeys) }
+  const isSupportedLanguage = Object.keys(availableTransations).some(lang => lang === language)
```

Comment:
Minor: I think we should try to distinguishing between `lang` and `language`. maybe rename `lang` to `aSupportedLanguage`